apps/ai-content-generator/src/configs/features/featureConfig.tsx

```diff
@@ -15,33 +15,33 @@ enum AIFeature {
 
 const featureConfig: FeatureConfig = {
   [AIFeature.TITLE]: {
-    title: 'Generate a title',
+    dialogTitle: 'Generate a title',
     buttonTitle: 'Title',
-    helpText: '',
+    description: 'generate a title',
     prompt: titlePrompt,
   },
   [AIFeature.CONTENT]: {
-    title: 'Generate content',
+    dialogTitle: 'Generate content',
     buttonTitle: 'Content',
-    helpText: '',
+    description: 'generate content',
     prompt: contentPrompt,
   },
   [AIFeature.TRANSLATE]: {
-    title: 'Language translation',
+    dialogTitle: 'Language translation',
     buttonTitle: 'Translate',
-    helpText: '',
+    description: 'translate',
     prompt: translatePrompt,
   },
   [AIFeature.SEO_KEYWORDS]: {
-    title: 'Generate SEO keywords',
+    dialogTitle: 'Generate SEO keywords',
     buttonTitle: 'SEO Keywords',
-    helpText: '',
+    description: 'generate SEO Keywords',
     prompt: seoKeywordsPrompt,
   },
   [AIFeature.SEO_DESCRIPTION]: {
-    title: 'Generate an SEO description',
+    dialogTitle: 'Generate an SEO description',
     buttonTitle: 'SEO description',
-    helpText: '',
+    description: 'generate an SEO description',
```

Comment:
`dialogTitle` is explicit now, and added the name of the other field as a `description` field. What do you think? I really am having trouble thinking of another name for it 😅

Reply:
What do you think about `invalidFormSuffix`? (or if I forgot about a place where it wasn't just used for an invalid spot, just `messageSuffix`